README.md

```diff
@@ -20,41 +21,38 @@ Conversations: Manage all your customer queries coming from the live chat plugin
 
 Dashboard: A powerful dashboard to see, analyze and act upon your customer conversation data. Helps you analyze the performance of support agents as well.
 
-FAQs: Create Frequently Asked Questions (FAQs) to cater to generic and recurring customer queries. Your customers will be able to directly access FAQs in chat.
+Helpcenter: Create your knowledge base and deploy on a dedicated page to cater to generic and recurring customer queries. Your customers will also be able to directly access FAQs in chat.
 
-Mailbox: A simple and powerful team inbox for managing, receiving and replying to all your customer support emails. 
+Mailbox: A simple and powerful team inbox for ticketing, managing, receiving and replying to all your customer support emails. 
 
 Integrations: Easily move data between Kommunicate and your other favorite apps. Integrate your favorite CRM, knowledge base software and other apps.
 
-Conversation Routing: Select routing rules for incoming conversations for both your agents and bots. Choose between a round robin assignment or to notify all.
+Conversation Routing: Select routing rules for incoming conversations for both your agents and bots. Choose between automatic assignments or to notify all.
 
-Smart Actionable Rich Messaging: Leverage actionable rich messages using buttons, cards or lists to provide an exquisite support chat experience to your customers.
+Smart Rich Messaging: Leverage rich messages using buttons, cards, carousels, forms or lists to provide an exquisite support chat experience to your customers.
 
 Quick Replies: Quickly respond to generic user queries using Quick Replies. Easily create and manage templated messages from your dashboard.
 
-Welcome Messages: Create custom and conditional welcome messages based on whether your customer support agents are online or offline. 
-
 ### Example :
 
 https://jsfiddle.net/Kommunicate/wgLuLLbu/
 
 ### Getting Started :
 
-Create your account by [signing](https://www.kommunicate.io/?utm_source=github&utm_medium=readme&utm_campaign=web) up for Kommunicate. If you already have a Kommunicate account, log in to your account and go to Settings -> Install section and copy the script.
+Create your account by [signing](https://www.kommunicate.io/?utm_source=github&utm_medium=readme&utm_campaign=web) up for Kommunicate. If you already have a Kommunicate account, log in to your account and go to [Settings -> Install](https://dashboard.kommunicate.io/settings/install) section and copy the script.
 
 Or
 
-You can copy the below script and replace required parameters manually.
+You can copy the below script and replace the required parameters manually. Note: You will get your <APP_ID> in the [Install](https://dashboard.kommunicate.io/settings/install) section. 
 
-Script
 
 ```
```

Comment:
@parthsh Can you replace the older script with the script which is there in our documentation?
https://docs.kommunicate.io/docs/web-installation#script